python/lsst/pipe/tasks/make_direct_warp.py

```diff
@@ -265,7 +314,7 @@ def bgSubtracted(self) -> bool:
 
```

Comment:
Having a non-configurable maximum limit is somewhat intentional, to prevent too many noise realizations been generated accidentally. I'd rather just let the `numberOfNoiseRealizations` take any non-negative integer value than setting a configurable maximum limit.